src/django/api/models/moderation_event.py

```diff
@@ -0,0 +1,112 @@
+import uuid
+from django.db import models
+from api.models.facility.facility_claim import FacilityClaim
+
+
+class ModerationEvent(models.Model):
+    '''
+    Data that is needed for moderation queue.
+    '''
+    uuid = models.UUIDField(
```

Comment:
1. Could you please add a description for this column?
2. I recommend making it the primary key. Do you have any objections to that?

Reply:
As for `uuid` vs `id`, we decided to keep `id` for now (re-check record from API Design sync 2024/10/15 starting from `1:46:13` ). One thing I've added for it is a description and make it indexing in the DB.

Reply:
@VadimKovalenkoSNF So we will use `id` and `uuid` in Moderation Event table?

Reply:
> So we will use id and uuid in Moderation Event table?

Yes, but only before migration to Kafka. After that we will use only `uuid`. Pls, refer to video record for more info.

Reply:
I personally don't see the value in using the ID, but OK